packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4585,14 +4585,11 @@ export class ContainerRuntime
 	 */
 	private reSubmitBatch(
 		batch: PendingMessageResubmitData[],
-		batchId: BatchId,
-		{ staged, squash }: { staged: boolean; squash: boolean },
+		{ batchId, staged, squash }: PendingBatchResubmitMetadata,
 	): void {
 		this.batchRunner.run(() => {
 			for (const message of batch) {
-				// Note: once squashing is widely supported, squashing even for orderSequentially should be fine.
-				// See remarks on orderSequentiallyRunner for more details.
-				this.reSubmit(message, this.orderSequentiallyRunner.running ? false : squash);
```

Comment:
do we even need orderSequentiallyRunner anymore? i would also consider explicting setting squash to false in order sequentially controlled staging mode. it will current inherit the false from the default, but if we change that, order sequentially should probably stay the way it is

Reply:
correct, we don't. I made that change somewhere but lost it in some merging I guess. Will fix :)